### DISCUSSION TITLE
read.me file style

### DISCUSSION THREAD

**Discussion by HossamMuhammedOmar** [community/community#32942] [Other Feature Feedback, Questions, & Ideas]
**Hello there!**
I believe it would be great to include additional style options in the readme file.
like text colour and font size (In addition to h1 or #).
Is this something that might be added to GitHub in the future?

**Comment by ghost** [comment]
Hi HossamMuhammedOmar! [Mdxjs](https://mdxjs.com/) is *Markdown for the component era* and has those features you request.

#### Example.mdx
```javascript
import {Chart} from './snowfall.js'
export const year = 2018

# Last year’s snowfall

In {year}, the snowfall was above average.
It was followed by a warm spring which caused
flood conditions in many of the nearby rivers.

<Chart year={year} color="#fcb32c" />
```
